src/Examples/UnitModels/Operations/translator_testing.ipynb

```diff
@@ -0,0 +1,500 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Translator Unit Model with Modular Property Package\n",
+    "\n",
+    "![](translator.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the Translator unit model in IDAES\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this tutorial, we will be transforming an inlet stream that uses the Benzene-Toluene Ideal property package to an outlet stream that uses a user-created configuration file. The state variables for each stream are as follows:\n",
```

Comment:
An explanation of when and why you would want to use a Translator block is important here, as it explains why we need a second property package with different state variables in the example.